cli/src/commands/diffedit.rs

```diff
@@ -42,8 +42,7 @@ use crate::ui::Ui;
 /// rebased on top as usual, which may result in conflicts.
 ///
 /// See `jj restore` if you want to move entire files from one revision to
-/// another. See `jj squash -i` or `jj unsquash -i` if you instead want to move
-/// changes into or out of the parent revision.
+/// another. For moving changes between revisions, see `jj squash -i`.
```

Comment:
Nit: Add the `jj diffedit --restore-descendants` case here.

Reply:
This is `jj diffedit` documentation, and `--restore-descendants` is mentioned in the paragraph just before this one. Isn't that sufficient?

> Unless `--restore-descendants` is used, descendants will be rebased on top as usual, which may result in conflicts.

Reply:
Oops, then its fine. My mistake.